src/screens/ContactScreen/index.js

```diff
@@ -16,8 +17,10 @@ import {Context} from '../../context';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
 import {Loading} from '../../components';
 import {Search} from './elements';
-import {userPopulate} from '../../service/users';
 import {withInteractionsManaged} from '../../components/WithInteractionManaged';
+import {ANONYMOUS} from '../../hooks/core/constant';
+import DiscoveryRepo from '../../service/discovery';
+import DiscoveryAction from '../../context/actions/discoveryAction';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `userPopulate` and addition of `ANONYMOUS`, `DiscoveryRepo`, and `DiscoveryAction` imports seem to be part of the refactoring process to handle anonymous chats. Ensure that these modules are correctly implemented and exported in their respective files.
```diff
- import {userPopulate} from '../../service/users';
+ import {withInteractionsManaged} from '../../components/WithInteractionManaged';
+ import {ANONYMOUS} from '../../hooks/core/constant';
+ import DiscoveryRepo from '../../service/discovery';
+ import DiscoveryAction from '../../context/actions/discoveryAction';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->